src/EventListener/CacheControlListener.php

```diff
@@ -210,16 +210,4 @@ private function setExtraCacheDirectives(Response $response, array $controls, $o
             }
         }
     }
-
-    /**
-     * Decide whether to even look for matching rules with the current request.
-     *
-     * @param Request $request
-     *
-     * @return bool True if the request is safe and headers can be set
-     */
-    protected function isRequestSafe(Request $request)
```

Comment:
imo we don't need this method anymore when there is no BC method_exists stuff anymore